landing-page/content/common/spark-quickstart.md

```diff
@@ -0,0 +1,379 @@
+---
+url: spark-quickstart
+toc: true
+aliases:
+    - "spark-quickstart"
+---
+<!--
+ - Licensed to the Apache Software Foundation (ASF) under one or more
+ - contributor license agreements.  See the NOTICE file distributed with
+ - this work for additional information regarding copyright ownership.
+ - The ASF licenses this file to You under the Apache License, Version 2.0
+ - (the "License"); you may not use this file except in compliance with
+ - the License.  You may obtain a copy of the License at
+ -
+ -   http://www.apache.org/licenses/LICENSE-2.0
+ -
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+ -->
+
+# Getting Started With Spark and Iceberg
+
+Spark is currently the most feature-rich compute engine for Iceberg operations. 
+We recommend you to get started with Spark to understand Iceberg concepts and features with examples.
```

Comment:
Do we need to say either of these things? Seems like it could get out of date quickly. Plus, we don't want Iceberg to steer people toward a particular engine. I think it's enough that we make this quickstart the default, right?

Reply:
I've updated this to:

> This quickstart will get you up and running with an Iceberg and Spark environment, including sample code to
> highlight some powerful features. You can learn more about Iceberg's Spark runtime by checking out the [Spark](../docs/latest/spark-ddl/) section.